app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -56,3 +56,7 @@
     font-size: var(--fs-s);
   }
 }
+
+.full-height {
```

Comment:
Refactor (blocking): Use the [h-100](https://storybook.forem.com/?path=/story/utility-first-classes-height--h-100) CSS utility class.

Reply:
oh awesome - for some reason I thought the util class was `h-full` but that was giving me `100vh`